go.mod

```diff
@@ -47,4 +47,7 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.2
 )
 
-replace github.com/GoogleCloudPlatform/spark-on-k8s-operator => github.com/lyft/spark-on-k8s-operator v0.1.3
+replace (
+	github.com/GoogleCloudPlatform/spark-on-k8s-operator => github.com/lyft/spark-on-k8s-operator v0.1.3
```

Comment:
didn't Anmol get rid of this? https://github.com/flyteorg/flytepropeller/commit/a15b94e2529d7b77dd65ec58416dd9b447714b2d

can we remove it here too?

Reply:
Done